content/speakers.yaml

```diff
@@ -0,0 +1,4 @@
+- name: Eric Ponto
```

Comment:
as i mentioned in slack, the goal of these being in markdown is to support profile pages as a later follow-up feature